master/buildbot/status/web/builder.py

```diff
@@ -41,9 +41,17 @@ def force(self, req, builderNames):
         owner = self.getAuthz(req).getUsernameFull(req)
         schedulername = req.args.get("forcescheduler", ["<unknown>"])[0]
         if schedulername == "<unknown>":
-            defer.returnValue((path_to_builder(req, self.builder_status),
-                               "forcescheduler arg not found"))
-            return
+            try:
+                for sch in master.allSchedulers():
```

Comment:
@campos-ddc it seems that this feature is not available from the UI (being able to request running builds on selected builders).

Maybe just add this, before your logic that we know it will work when forcing a single builder
if len(builderNames) == 1: